Solution/HW05.Task1/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace HW05.Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string alphabet="";
+            for (int i = 122; i >=65; i--)
```

Comment:
what the magic numbers - 122, 65?

Reply:
In this space located english aplhabet with symbols, which i`ve deleted in the next lines.